src/main/java/com/commercebank/controller/AppointmentController.java

```diff
@@ -60,6 +60,11 @@ public AppointmentController(AppointmentDAO appointmentDAO, CustomerDAO customer
         return appointmentDAO.list();
     }
 
+    @RequestMapping(value = "/delete", method = RequestMethod.DELETE)
```

Comment:
Pass the id in the URL, like `/delete/{id}`, then you don't need the extra code in the DAO.